tests/series/arithmetic/complex/test_sub.py

```diff
@@ -0,0 +1,129 @@
+import numpy as np
+from numpy import typing as npt  # noqa: F401
+import pandas as pd
+from typing_extensions import assert_type
+
+from tests import check
+
+left = pd.Series([1j, 2j, 3j])  # left operand
+
+
+def test_sub_py_scalar() -> None:
+    """Test pd.Series[complex] - Python native scalars"""
+    i, f, c = 1, 1.0, 1j
```

Comment:
I think you could do:
```suggestion
    b, i, f, c = True, 1, 1.0, 1j
```
And then add the tests on `b` here, and similar in the other funcs.